BLE Indoor Positioning/src/test/java/com/nexenio/bleindoorpositioning/location/LocationTest.java

```diff
@@ -24,7 +23,7 @@
 
     public static final Location NEW_YORK_CITY = new Location(40.7127753, -74.0059728);
     public static final Location BERLIN = new Location(52.5200066, 13.404954);
-    public static final int DISTANCE_NYC_BERLIN = 6381 * 1000;
+    public static final int DISTANCE_NYC_BERLIN = 6385 * 1000;
```

Comment:
Changing static test parameters is often used to mask the result of a fault (Locations did not change). Why did this parameter change?

Reply:
Because i could not find any source supporting 6381 and found evidence from multiple sources that the new distance is correct

Reply:
What I meant was not which distance may be the correct one, but that the location is still the same. Therefore the calculation must have changed to result in a new distance. Doing so can be fine, but constants like 'DISTANCE_NYC_BERLIN' should normally not change as they do not change in real life. 
In this example a difference of a few km is not a problem of cause, as there are different sources.